docs/jwst/extract_2d/main.rst

```diff
@@ -27,7 +27,7 @@ the 2D cutouts from the dispersed image.
 
 Algorithm
 ---------
-The step is currently applied only to NIRSpec Fixed Slit, NIRSPEC MSA,
+The step is currently applied only to NIRSpec Fixed Slit, NIRSPEC MSA, NIRSPEC TSO,
 NIRCAM GRISM and NIRISS WFSS observations.
```

Comment:
Should update "NIRCAM GRISM" to "NIRCAM WFSS", now that we're also changing the EXP_TYPE to "NRC_WFSS". I think that also helps to distinguish it from NIRCam TS Grism mode.